account_entries_report_group_by_ref/account_entries_report_view.xml

```diff
@@ -1,17 +1,17 @@
 <?xml version='1.0' encoding='utf-8'?>
 <openerp>
     <data>
-        
+
         <record id="view_account_entries_report_search_origin_inh" model="ir.ui.view">
             <field name="name">view.account.entries.report.search.origin.inh</field>
             <field name="model">account.entries.report</field>
             <field name="inherit_id" ref="account.view_account_entries_report_search"/>
             <field name="arch" type="xml">
-                <xpath expr="//group[@string='Group By...']/filter[@string='Fiscal Year']" position="after">
+                <xpath expr="//group[@string='Group By']/filter[@string='Company']" position="after">
```

Comment:
@nhomar `Fiscal Year` is not in [ view_account_entries_report_search v.8.0] (https://github.com/odoo/odoo/blob/8.0/addons/account/report/account_entries_report_view.xml#L84) I can use [`Entries Month`] (https://github.com/odoo/odoo/blob/8.0/addons/account/report/account_entries_report_view.xml#L91) instead.